xml/System.Data.Common/DbBatch.xml

```diff
@@ -401,10 +475,19 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="behavior">One of the enumeration values that specifies options for batch execution and data retrieval.</param>
```

Comment:
Something weird happened with this parameter. Anyway, it will be fixed in https://github.com/dotnet/dotnet-api-docs/pull/7909.